services/src/northbound/src/main/java/org/openkilda/northbound/service/BasicService.java

```diff
@@ -81,4 +87,24 @@ default InfoData validateInfoMessage(final Message requestMessage,
         }
         return data;
     }
+
+    /**
+     * Validates switch existing.
+     *
+     * @param correlationId the correlation id.
+     * @param data list with source and destination switches.
+     */
+    default void validateSwitches(String correlationId, List<SwitchInfoData> data, FlowPayload flowPayload) {
+        List<SwitchInfoData> switches = data.stream().filter(switchData ->
+                switchData.getSwitchId().toString().equals(flowPayload.getSource().getSwitchDpId().toString())
+                        || switchData.getSwitchId().toString()
+                        .equals(flowPayload.getDestination().getSwitchDpId().toString()))
+                .collect(Collectors.toList());
+        if (switches.size() < 2) {
+            String errorMessage = "Source or destination switch is not exists";
+            logger.error(errorMessage);
+            throw new MessageException(correlationId, System.currentTimeMillis(), DATA_INVALID, errorMessage,
+                    "Could not create flow");
```

Comment:
'Couldn't create **a** flow' or 'Unable to create **a** flow' - whatever you want.